crates/autopilot/src/arguments.rs

```diff
@@ -147,6 +148,10 @@ pub struct Arguments {
     /// The time in seconds between new blocks on the network.
     #[clap(long, env, value_parser = shared::arguments::duration_from_seconds)]
     pub network_block_interval: Option<Duration>,
+
+    /// Defines which strategies to apply when updating the `surplus_fee` of limit orders.
+    #[clap(long, env, use_value_delimiter = true)]
+    pub quoting_strategies: Vec<QuotingStrategy>,
```

Comment:
Why Vec instead of a single strategy?

Reply:
I wanted to be able to compose different strategies in the future. For example I have in mind to prefer updating orders which have an achievable price instead of going by the `surplus_fee_tiemstamp`.
Eventually this should get dropped entirely when we tested each strategy and decided on whether to keep it or not.

Reply:
Feature flags for testing how different strategies

Reply:
Here is the issue: https://github.com/cowprotocol/services/issues/1168